INSTALL.rst

```diff
@@ -123,6 +148,12 @@ We build assets using `npm`. Make sure you have installed it system wide.
     still setup a per-user npm modules installation that will allow you to
     install/remove modules as normal user. You can find more info `in the npm
     docs here`_.
+    
+    If you want to install npm packages directly in your virtualenv
+    just add `NPM_CONFIG_PREFIX=$VIRTUAL_ENV`
+    in the postactivate file of your virtualenv folder and you will be able to run
+    `npm install -g node-sass@3.8.0 clean-css requirejs uglify-js`
+    from inside your virtual environment.
```

Comment:
Oh, I like this! I was actually polluting my global config with a `NPM_CONFIG_PREFIX=$HOME/.npm`.